deploy-manage/deploy/elastic-cloud/cloud-hosted.md

```diff
@@ -50,10 +51,159 @@ $$$faq-where$$$
+* [Create a deployment](/deploy-manage/deploy/elastic-cloud/create-an-elastic-cloud-hosted-deployment.md) - Get up and running very quickly. Select your desired configuration and let Elastic deploy Elasticsearch, Kibana, and the Elastic products that you need for you. In a deployment, everything works together, everything runs on hardware that is optimized for your use case.
+* [Connect your data to your deployment](/manage-data/ingest.md) - Ingest and index the data you want, from a variety of sources, and take action on it.
+
+**Adjust the capacity and capabilities of your deployments for production**
+
+There are a few things that can help you make sure that your production deployments remain available, healthy, and ready to handle your data in a scalable way over time, with the expected level of performance. We’ve listed these things for you in [Preparing for production](/deploy-manage/deploy/elastic-cloud/cloud-hosted.md).
+
+**Secure your environment**
+
+Control which users and services can access your deployments by [securing your environment](/deploy-manage/users-roles/cluster-or-deployment-auth.md). Add authentication mechanisms, configure [traffic filtering](/deploy-manage/security/traffic-filtering.md) for private link, encrypt your deployment data and snapshots at rest [with your own key](/deploy-manage/security/encrypt-deployment-with-customer-managed-encryption-key.md), manage trust with Elasticsearch clusters from other environments, and more.
+
+**Monitor your deployments and keep them healthy**
+
+{{ech}} provides several ways to monitor your deployments, anticipate and prevent issues, or fix them when they occur. Check [Monitoring your deployment](/deploy-manage/monitor/stack-monitoring.md) to get more details.
+
+**And then?**
+
+Now is the time for you to work with your data. The content of the {{ech}} section helps you get your environment up and ready to handle your data the way you need. You can always adjust your deployments and their configuration as your usage evolves over time.
+
+To get the most out of the solutions that the Elastic Stack offers, [log in to {{ecloud}}](https://cloud.elastic.co) or [browse the documentation](https://www.elastic.co/docs).
+
+## More about {{ech}} [ec-about]
+
+Find more information about {{ech}} on the following pages:
+
+* [Subscription Levels](/deploy-manage/license.md)
+* [Version Policy](/deploy-manage/deploy/elastic-cloud/available-stack-versions.md)
+* [{{ech}} Hardware](https://www.elastic.co/guide/en/cloud/current/ec-reference-hardware.html)
+* [{{ech}} Regions](https://www.elastic.co/guide/en/cloud/current/ec-reference-regions.html)
+* [Service Status](/deploy-manage/cloud-organization/service-status.md)
+* [Getting help](/troubleshoot/index.md)
+* [Restrictions and known problems](/deploy-manage/deploy/elastic-cloud/restrictions-known-problems.md)
+
+:::{dropdown} {{ech}} FAQ
```

Comment:
I like the dropdown, but it's not rendering super well

Reply:
Yup, this is a bit massive to stay on the page as is, will leave it as is for now. It may be simplified or taken out to a different page in the future.